appendices/VK_EXT_physical_device_drm.adoc

```diff
@@ -25,12 +25,17 @@ Unlike the EGL extension, this extension does not expose a string containing
 the name of the device file and instead exposes device minor numbers.
 
 DRM defines multiple device node types.
-Each physical device may have one primary node and one render node
-associated.
-Physical devices may have no primary node (e.g. if the device does not have
-a display subsystem), may have no render node (e.g. if it is a software
-rendering engine), or may have neither (e.g. if it is a software rendering
-engine without a display subsystem).
+Each physical device can only be associated with a single underlying DRM
+device, which may have one primary node and one render node.
+
+Physical devices may have no primary node (e.g. it is not associated with a
```

Comment:
I don't think that case is possible, actually. A render node always has a primary node.

Reply:
I agree, I think the only case is the same as a software implementation which does not use DRM. Is it better to switch this text to something like "it always has the primary node if it is associated with a DRM device"?